packages/@cagov/accordion/src/index.js

```diff
@@ -37,48 +49,48 @@ export class CaGovAccordion extends window.HTMLElement {
ng:0 .5rem 0 0!important}.accordion-title h2,.accordion-title h3,.accordion-title h4{padding:0!important;margin-top:0!important;margin-bottom:0!important;font-family:Roboto,sans-serif;font-size:1.2rem!important;font-weight:700;color:#1f2574;text-align:left!important}button.card-header{display:flex;justify-content:left;align-items:center;padding:1rem .75rem;background-clip:border-box;background-color:#fff;border:none;border-bottom:1px solid #1f2574;border-top:1px solid #1f2574;border-radius:0!important;margin-top:-1px}button.card-header:hover{background-color:#f2f5fc}button.card-header:focus{outline:2px solid #ffcf44;outline-offset:-2px}.prog-enhanced .accordion-alpha .plus-munus{width:2.125rem;height:2.125rem;border:2px solid #ff8000;border-radius:50%;padding:0;overflow:hidden;transition:all .3s ease;margin-left:1rem;color:#ff8000;align-self:flex-start}.prog-enhanced .accordion-alpha .plus-munus agov-plus .accordion-icon{display:block}.prog-enhanced .accordion-alpha .plus-munus cagov-minus .accordion-icon{display:none}.prog-enhanced .accordion-alpha-open{border-bottom-color:#fff!important}.prog-enhanced .accordion-alpha-open cagov-plus .accordion-icon{display:none}.prog-enhanced .accordion-alpha-open cagov-minus .accordion-icon{display:block!important}.dark-accordion-bg>.container.dark-accordion-first{padding-top:3rem;margin-top:3rem;padding-bottom:0;margin-bottom:0}.dark-accordion-bg>.container{padding-top:3rem;margin-top:3rem;padding-bottom:3rem;margin-bottom:3rem}.dark-accordion-sibling>.container{padding-top:0;padding-bottom:0;margin-top:0;margin-bottom:0}.dark-accordion-sibling+.dark-accordion-bg>.container{padding-top:0;margin-top:0;padding-bottom:3rem;margin-bottom:3rem}.dark-accordion-bg .card{background-color:#1f2574}.dark-accordion-bg .card .accordion-alpha{background-color:#1f2574;border-bottom:1px solid #4c5190;border-top:1px solid #4c5190}.dark-accordion-bg .card .accordion-alpha.accordion-alpha-open{border-bottom-color:#1f2574!important}.dark-accordion-bg .card .accordion-alpha:focus,.dark-accordion-bg .card .accordion-alpha:hover{background-color:#003d9d}.dark-accordion-bg .card .accordion-alpha .accordion-title{color:#fff}.dark-accordion-bg .card .accordion-alpha .accordion-title h2,.dark-accordion-bg .card .accordion-alpha .accordion-title h3,.dark-accordion-bg .card .accordion-alpha .accordion-title h4{color:#fff}.dark-accordion-bg .card .card-container{color:#fff}.dark-accordion-bg .card .card-container[aria-hidden=false]{border-bottom:1px solid #4c5190}.dark-accordion-bg .card .card-container h2,.dark-accordion-bg .card .card-container h3,.dark-accordion-bg .card .card-container h4,.dark-accordion-bg .card .card-container h5{color:#fff}.bg-darkblue .dark-accordion-bg>.container.dark-accordion-first{padding-top:1rem;margin-top:0;padding-bottom:0;margin-bottom:0}.fill-orange{fill:#ff8000!important}.card-body{padding:.55rem .45rem .55rem .75rem;color:#000;line-height:1.5}.card-body p{font-family:Roboto,sans-serif;font-weight:300}
```

Comment:
I'd like to add a scss file into the component package, add a sass build step into the package build process and import the generates css file here. We can do that in a subsequent ticket and hopefully follow a similar pattern in other components